src/app/widgets/bouncy-loader/bouncy-loader.component.html

```
@@ -0,0 +1,8 @@
+<div class="container">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)